etl/steps/data/garden/artificial_intelligence/2024-07-16/cset.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import owid.catalog.processing as pr
+import pandas as pd
+from owid.catalog import Dataset, Table
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def add_world(tb: Table, ds_regions: Dataset) -> Table:
+    # Create a deep copy of the input table to avoid modifying the original data
+    tb_with_regions = tb.copy()
+
+    # List of members representing different regions CSET and use these for World aggregation
+    members = [
+        "North America",
+        "Europe",
+        "Asia Pacific",
+        "Africa",
+        "Latin America and the Caribbean",
+        "Oceania",
+        "OECD",
+        "Five Eyes (Australia, Canada, New Zealand, UK, and the US)",
```

Comment:
Maybe there is some double-counting here? E.g. Australia is in there twice, US is in a few different groups too.

Reply:
ah sorry I am actually excluding these for the World aggregation despite the comment above :) I've changed that now!